test/integration/PartnerRegistrar.spec.ts

```diff
@@ -256,6 +268,29 @@ describe('New Domain Registration', () => {
     );
   });
 
+  it('Should register a new domain for a partnerOwnerAccount with 0 minCommitmentAge', async () => {
```

Comment:
The title of the test and what the test does seems to be inconsistent

Reply:
Yes thanks